Bencodex/Types/Dictionary.cs

```diff
@@ -54,6 +54,12 @@ public bool TryGetValue(IKey key, out IValue value)
 
         public IEnumerable<IValue> Values => Value.Values;
 
+        public static implicit operator Dictionary(ImmutableDictionary<IKey, IValue> dictionary)
```

Comment:
```suggestion
        public static implicit operator Dictionary(IImmutableDictionary<IKey, IValue> dictionary)
```

Reply:
it will occurs *[CS0552]*.
It's restricted to create user-defined conversion to or from an interface.

[CS0552]: https://docs.microsoft.com/ko-kr/dotnet/csharp/language-reference/compiler-messages/cs0552

Reply:
Hmm… C# disallows to declare implicit/explicit conversion from/to an interface.  Then IMHO `Dictionary` having conversion operator would not be that useful.

Reply:
Yeah, I agree. So I removed them. :grin: